src/modules/claim/claim.service.ts

```diff
@@ -51,10 +51,16 @@ export class ClaimService {
   }
 
   private initializeExchangeListener() {
-    this.nats.connection.subscribe(`*.${NATS_EXCHANGE_TOPIC}`, async data => {
-      this.logger.debug(`Got message ${data}`);
-      await this.claimQueue.add('save', data);
-    });
+    this.nats.connection.subscribe(
+      `*.${NATS_EXCHANGE_TOPIC}`,
+      {
+        queue: 'iam_cache_server_exchange_topic',
+      },
+      async data => {
+        this.logger.debug(`Got message ${data}`);
+        await this.claimQueue.add('save', data);
+      },
+    );
```

Comment:
@Passerino with our messaging changes, I don't think we expect the cache-server to be receiving any messages at this moment, right? All communication **to** the cache-server is done via HTTP, I believe. Perhaps we can remove this code?

Reply:
@Passerino  Ah, nevermind, I see you have commented it out in your PR already, so I think we can remove it in your PR 😄  https://github.com/energywebfoundation/iam-cache-server/pull/205/files